code/src/components/Header.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import styled from 'styled-components'
+import moment from 'moment'
+import { useDispatch, useSelector } from 'react-redux'
+import { tasks } from 'reducers/tasks'
+
+export const Header = () => {
+
+  const dispatch = useDispatch()
+
+  // To get all tasks
+  const allTasks = useSelector(store => store.tasks.items)
+
+  // To get tasks with completed status true
+  const completedTasks = allTasks.filter(task => task.completed === true)
+
+  // Function to clear all todos
+  const handleClearTasks = () => {
+    dispatch(tasks.actions.removeAll())
+  }
+
+  return (
+    <WrapperHeader>
+      <WrapperLeft>
+        <Heading>My todos</Heading>
+        <Text>{moment().format("MMMM Do")}</Text>
+      </WrapperLeft>
+      <WrapperRight>
+        <Text>{completedTasks.length}/{allTasks.length} done</Text>
+        <ClearButton onClick={handleClearTasks}>Clear all</ClearButton>
+      </WrapperRight>
+    </WrapperHeader>
+  )
+
+}
+
+const WrapperHeader = styled.section`

```

Comment:
This is more of a preference thing, but generally I'd recommend having all the styles defined before the component itself. The way the JS compiles, it's not a problem, but technically, you should define things _before_ they're used in the file, so using these components in the `return` of your component and then defining them below is a bit backwards.

Reply:
Will apply that in next project!